src/uuid/model/entity_revision/course_revision.rs

```diff
@@ -6,21 +6,21 @@ use super::abstract_entity_revision::AbstractEntityRevision;
 #[serde(rename_all = "camelCase")]
 pub struct CourseRevision {
     title: String,
-    content: String,
+    description: String,
```

Comment:
This should still be `content` to not introduce a breaking change.

Reply:
ah I see, just change the field it expects from the db then.

Reply:
The contract tests catched that btw, yay 🎉